docs/java-rest/high-level/watcher/get-watch.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[java-rest-high-x-pack-watcher-get-watch]]
+=== Get Watch API
```

Comment:
There were a couple changes made to the HLRC docs set-up:
- It's now common to define and use variables like `:api:` instead of hard-coding names.
- With these variables defined, you can use `include::../execution.asciidoc[]` to automatically include the 'asynchronous execution' section.

Here's an example that has been updated to the new format: https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/java-rest/high-level/watcher/ack-watch.asciidoc